src/content/enterprise/administration/volume-snapshots.mdx

```diff
@@ -309,4 +310,6 @@ spec:
 </TabItem>
 </Tabs>
 
-As part of the creation of the persistent volume claim, Okteto will import the specified snapshot into your cluster using a `VolumeSnapshotContent` and will set the source of your persistent volume claim to this [VolumeSnapshotContent](https://kubernetes.io/docs/concepts/storage/volume-snapshots/), ready to be used by your development environment.
\ No newline at end of file
+As part of the creation of the persistent volume claim, Okteto will import the specified snapshot into your cluster using a `VolumeSnapshotContent` and will set the source of your persistent volume claim to this [VolumeSnapshotContent](https://kubernetes.io/docs/concepts/storage/volume-snapshots/), ready to be used by your development environment.
+
+Allowing volume snapshots from snapshot id is enabled by default, you can disable this feature by changing the config `allowIDAnnotation: false` at the cluster configuration `volumeSnappshots`.
```

Comment:
Why would someone want to disable this? Can we specify the scenario this enables?

Reply:
Sure, added. The case where OE customer wants to prevent the usage of snapshots-id as source for PVCs